src/lib/methods/method.ts

```diff
@@ -16,4 +19,23 @@ export abstract class Method implements QueryBuilder {
 	 * Builds and returns the raw DynamoDB query object.
 	 */
 	abstract buildRawQuery(): any;
+
+	/**
+	 * Configure the retry policy
+	 *
+	 * @param opts - Retry configuration options.
```

Comment:
```suggestion
	 * @param retries - Retry configuration options.
```

Reply:
If you want, you can actually work with overrides.

```ts
retry(retries: number);
retry(options: RetryOptions);

retry(options: number | RetryOptions) {
	this.retries = options;

	return this;
}
```

Only the first two will end up in intellisense so you can document them differently. It's just a TS feature, so feel free to use it. But if you want, you may keep the current implementation.

Reply:
When trying to implement your suggestion, the linter is complaining. Can I add an exception? 

`ERROR: 34:8  unified-signatures      These overloads can be combined into one signature taking `number | RetryOptions`.`

Reply:
Yup, just disable it :).